### PR TITLE
SAMA-14 implemented the page of information about chat

### PR DESCRIPTION
**Pull request by Oleksandr1414**
* added a page for group info
* added a page for another user to view

### CODE REVIEW

File: src/components/screens/info/UserGuestProfile.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import { useSelector } from "react-redux";
+
+import "./../../../styles/pages/UserProfile.css";
+
+import { ReactComponent as BackBtn } from "./../../../assets/icons/chatForm/BackBtn.svg";
+import { ReactComponent as EmailIcon } from "./../../../assets/icons/userProfile/EmailIcon.svg";
+import { ReactComponent as PhoneIcon } from "./../../../assets/icons/userProfile/PhoneIcon.svg";
+import { ReactComponent as UserLoginIcon } from "./../../../assets/icons/userProfile/UserLoginIcon.svg";
+
+export default function UserGuestProfile() {
```
Comment (IgorKhomenko) [inline src/components/screens/info/UserGuestProfile.js:14]: UserGuestProfile -> UserProfile

File: src/components/screens/info/ChatInfoPage.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        dispatch(removeChat(selectedCID));
+        history.navigate("/main");
+      } catch (error) {
+        showCustomAlert(error.message, "warning");
+      }
+    }
+  };
+  // ʌʌ  Delete chat block  ʌʌ //
+
+  // vv  Remove user block  vv //
```
Comment (IgorKhomenko) [inline src/components/screens/info/ChatInfoPage.js:57]: remove this code

File: src/components/screens/info/ChatInfoPage.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    return selectedConversation.participants.map((el) => {
+      const u = participants[el];
+      if (!u) {
+        return null;
+      }
+
+      const isCurrentUser = u._id === userInfo._id;
+      // const isOwner = userInfo._id === selectedConversation.owner_id.toString();
+      return (
+        <div
```
Comment (IgorKhomenko) [inline src/components/screens/info/ChatInfoPage.js:87]: can we move it to separate component?

File: src/services/autoLoginService.js [hunk 4]
```diff
@@ -39,8 +39,22 @@ class AutoLoginService {
...
 
         currentPath &&
-          store.dispatch(setSelectedConversation({ id: currentPath.slice(1) }));
+          store.dispatch(
+            setSelectedConversation({ id: currentPath.split("/")[0].slice(1) })
+          );
         store.dispatch(setUserIsLoggedIn(true));
+
+        const token = localStorage.getItem("sessionId");
+        if (token && token !== "undefined") {
```
Comment (IgorKhomenko) [inline src/services/autoLoginService.js:48]: do we really need it?

we already set it `      localStorage.setItem("sessionId", userToken);` above, 
so it always will be non-empty

**Review by ccvlad — APPROVED** [review]
lgfm

File: src/components/screens/info/ChatInfoPage.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+  const participants = useSelector(selectParticipantsEntities);
+  const selectedConversation = useSelector(getConverastionById);
+  const selectedCID = selectedConversation?._id;
+
+  const userInfo = localStorage.getItem("sessionId")
+    ? jwtDecode(localStorage.getItem("sessionId"))
+    : null;
+
+  const prevPageLink = useMemo(() => {
```
Comment (DSuvoZera) [inline src/components/screens/info/ChatInfoPage.js:31]: get rid of useMemo. no profit really

**Review by DSuvoZera — APPROVED** [review]

**Pull request merged**